### PR TITLE
Tweaks to contributing guide

### PR DESCRIPTION
**Pull request by karenmajewicz**
A few more updates following up on #1462 and #1460

### CODE REVIEW

**Comment by ewlarson** [comment]
Sorry! I thought you guys were done when Becky tagged me to review. Are these the final changes?

**Comment by karenmajewicz** [comment]
Yeah, we accidentally tagged you before we were done. We are done now.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ Issues are used in the GeoBlacklight community as a convenient way to document b
 Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to add an issue for it in the [issue tracker](https://github.com/geoblacklight/geoblacklight/issues). 
 
  - Make sure you have a [GitHub account](https://github.com/signup/free)
- - Submit a [GitHub issue](./issues) by:
-    - Clearly describing the issue
-    - Provide a descriptive summary
-    - Explain the expected behavior
-    - Explain the actual behavior
-    - Provide steps to reproduce the actual behavior
+ - Submit a [GitHub issue](./issues) that:
```
Comment (ewlarson) [inline CONTRIBUTING.md:19]: This link 404s.

Comment (ewlarson) [reply CONTRIBUTING.md:19]: Probably want `https://github.com/geoblacklight/geoblacklight/issues` instead.

**Review by ewlarson — CHANGES_REQUESTED** [review]
I like it. One little broken link to fix here.

**Review by ewlarson — APPROVED** [review]
👍 // Thanks again!

**Pull request merged**